ui/src/app/components/object-detail/object-detail.component.html

```diff
@@ -753,7 +742,7 @@ <h3 class="title">
           "
           mat-raised-button
           color="primary"
```

Comment:
Modify the SQL in SQL tab after interleaving property is removed

Reply:
Updated